superset-frontend/src/dashboard/components/nativeFilters/FiltersConfigModal/FiltersConfigForm/FiltersConfigForm.tsx

```diff
@@ -1035,6 +1035,7 @@ const FiltersConfigForm = (
                     >
                       <DateFilterControl
                         name="time_range"
+                        endpoints={['inclusive', 'exclusive']}
```

Comment:
Do you think it would make sense placing `DEFAULT_TIME_RANGE_ENDPOINTS = ['inclusive', 'exclusive']` under `src/constants.ts` and then using that here and on the other place?

Reply:
This seems more like the default value of `endpoints` in `DateFilterControl` than a global thing.